openlibrary/plugins/upstream/account.py

```diff
@@ -692,8 +692,11 @@ def GET(self):
 
     @require_login
     def POST(self):
+        expires = 3600 * 24 * 365
+        i = web.input(public_readlog="", safe_mode="")
         user = accounts.get_current_user()
-        user.save_preferences(web.input())
+        user.save_preferences(i)
+        web.setcookie('sfw', i.safe_mode, expires=expires)
```

Comment:
For readability, declare the variable just before it is used and use its name to help your reader understand that this is the number of seconds in a non-leap year...
```suggestion
        one_year = 3600 * 24 * 365
        web.setcookie('sfw', i.safe_mode, expires=one_year)
```